targets/drive/mobile/main.jsx

```diff
@@ -38,6 +38,8 @@ if (__DEVELOPMENT__) {
   require('preact/devtools')
 }
 
+window.handleOpenURL = require('drive/mobile/lib/handleOpenURL').default(hashHistory)
```

Comment:
How is the `handleOpenURL` name defined? Can it be changed to something more explicit?

Reply:
It is hardcoded in the plugin : I cannot rename the attribute of window. I can rename the lib into "deeplink". What do you think ?

Reply:
👍 Done